Criterion/Monad.hs

```diff
@@ -35,7 +42,12 @@ withConfig cfg (Criterion act) = withCP65001 $ do
 -- This is not currently thread-safe, but in a harmless way (we might
 -- call 'createSystemRandom' more than once if multiple threads race).
 getGen :: Criterion GenIO
-getGen = memoise gen createSystemRandom
+getGen = memoise gen
+#if defined(__GHCJS__)
+  create -- there is no /dev/urandom or GHCJS, let's just use fixed seed.
```

Comment:
Is there really no other source of randomness we could use here?

Reply:
One can play with `time`, I'm too lazy.